src/common/models/Projection.ts

```diff
@@ -258,45 +262,36 @@ export class Projection {
       this.icuHeadroomInfo?.metricSeries || this.dates.map(date => null);
 
     this.contractTracers = metricsTimeseries.map(row =>
-      row ? row.contactTracerCapacityRatio : null,
+      row?.contactTracerCapacityRatio ? row.contactTracerCapacityRatio : null,
     );
 
     this.caseDensityByCases = metricsTimeseries.map(row =>
-      row ? row.caseDensity : null,
+      row?.caseDensity ? row.caseDensity : null,
     );
     this.caseDensityRange = this.calcCaseDensityRange();
 
     this.currentCaseDensity =
-      metrics && !DISABLED_CASE_DENSITY.includes(this.fips)
+      metrics?.caseDensity && !DISABLED_CASE_DENSITY.includes(this.fips)
         ? metrics.caseDensity
         : null;
     this.currentDailyDeaths = lastValue(this.smoothedDailyDeaths);
 
-    const shortageStart =
-      summaryWithTimeseries.projections?.totalHospitalBeds?.shortageStartDate ||
-      null;
-    this.dateOverwhelmed =
-      shortageStart === null ? null : new Date(shortageStart);
-    if (
-      moment(this.dateOverwhelmed).diff(moment(), 'days') >
-      PROJECTIONS_TRUNCATION_DAYS
-    ) {
-      this.dateOverwhelmed = null;
-    }
-
-    this.currentCumulativeDeaths =
-      summaryWithTimeseries.actuals.cumulativeDeaths;
-    this.currentCumulativeCases =
-      summaryWithTimeseries.actuals.cumulativeConfirmedCases;
+    this.currentCumulativeDeaths = summaryWithTimeseries.actuals.deaths
+      ? summaryWithTimeseries.actuals.deaths
+      : null;
+    this.currentCumulativeCases = summaryWithTimeseries.actuals.cases
+      ? summaryWithTimeseries.actuals.cases
+      : null;
     this.currentContactTracerMetric =
-      metrics && !DISABLED_CONTACT_TRACING.includes(this.fips)
+      metrics?.contactTracerCapacityRatio &&
+      !DISABLED_CONTACT_TRACING.includes(this.fips)
         ? metrics.contactTracerCapacityRatio
         : null;
   }
 
   get currentContactTracers() {
     return (
-      CONTACT_TRACER_STATE_OVERRIDES[this.stateName] ||
+      CONTACT_TRACER_STATE_OVERRIDES[this.state] ||
```

Comment:
Should we drop this override since it's not used, and it won't work right unless we override it in the backend and the frontend anyway, which is 🤮 ?

Reply:
Dropped!